yogstation/code/modules/mob/living/living.dm

```diff
@@ -0,0 +1,17 @@
+/mob/living/proc/get_blood_state() // Returns how our good or bad our blood situation is, using the blood enums in Yogstation-TG\code\__DEFINES\mobs.dm
+	if(!initial(blood_volume))
+		return BLOOD_SAFE // I... guess we're fine?
+	
+	switch(blood_volume / initial(blood_volume))
```

Comment:
Why decide by the initial value?

Reply:
Because originally this switch was changed to use the _MULTI stuff. I'll go ahead & fix it.